### DISCUSSION TITLE
Single types via GUI

### DISCUSSION THREAD

**Discussion by ndt080** [pocketbase/pocketbase#3428] [Q&A] [answered]
Is it possible to create a Single type via GUI? We are talking about the type of queries that would return not a collection, but a single element. Using pocketbase as a framework is not suitable, since the data must be edited by the administrator through the GUI.

**Comment by ganigeorgiev** [comment, marked as answer]
Not at the moment (_this will be handled eventually in https://github.com/pocketbase/pocketbase/issues/159, but it is push aside for now_).

As a workaround, you can create a new collection with only 1 record and query just the record.

**Reply by ndt080** [reply]
I may have to do this

**Reply by ganigeorgiev** [reply]
Make sure to check the [Contributing note](https://github.com/pocketbase/pocketbase#contributing).

Note that even if you decide to work on it (_there was also already a PR for it in https://github.com/pocketbase/pocketbase/pull/2778_), I may not merge it at the moment (I'm planning some internal refactoring and I don't want to add new collection types for now).
